kuma/users/auth.py

```diff
@@ -29,18 +29,15 @@ def filter_users_by_claims(self, claims):
         return user_model.objects.filter(username=fxa_uid)
 
     def create_user(self, claims):
-        if (
-            not (subscriptions := claims.get("subscriptions"))
-            or settings.MDN_PLUS_SUBSCRIPTION not in subscriptions
-        ):
-            return None
         user = super().create_user(claims)
 
         self._create_or_set_user_profile(user, claims)
+        self.request.created = True
         return user
 
     def update_user(self, user, claims):
         self._create_or_set_user_profile(user, claims)
+        self.request.created = False
```

Comment:
super nit: You can omit completely and change the check in the CallbackView to
`if self.request.get("created") and not is_subscriber`